Arduino.mk

```diff
@@ -444,6 +444,9 @@ ifndef AVR_TOOLS_DIR
         ifdef SYSTEMPATH_AVR_TOOLS_DIR
             AVR_TOOLS_DIR = $(SYSTEMPATH_AVR_TOOLS_DIR)
             $(call show_config_variable,AVR_TOOLS_DIR,[AUTODETECTED],(found in $$PATH))
+            ifndef AVRDUDE_CONF
+                AVRDUDED_CONF := $(call dir_if_exists,/etc/avrdude/avrdude.conf)
```

Comment:
AVRDUDE**D** typo

is /usr/share/arduino/hardware/tools/avrdude.conf not just a symlink to /etc/avrdude.conf on fedora (like debian)? and is it really /etc/avrdude/avrude.conf or do you mean /etc/avrdude.conf ?